datadog_checks_dev/datadog_checks/dev/tooling/commands/metadata.py

```diff
@@ -0,0 +1,322 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import csv
+from collections import defaultdict
+from io import open
+
+import click
+from six import PY2, iteritems
+
+from .utils import CONTEXT_SETTINGS, abort, echo_failure, echo_warning
+from ..utils import get_metadata_file, get_metric_sources, load_manifest
+
+REQUIRED_HEADERS = {
+    'metric_name',
+    'metric_type',
+    'description',
```

Comment:
Sorry I just noticed this, but description is not actually mandatory: you can leave it empty and it is accepted by our pipeline. 
This will avoid having "placeholder" descriptions that don't really add value.
This probably should be a warning, as a description is strongly advised, but shouldn't make the task fail.

Reply:
should the doc here be changed as well to reflect that? https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#metadatacsv

Reply:
Yes it should. Very good catch!